src/firebase-db.ts

```diff
@@ -409,3 +414,53 @@ export const setLearnerPluginState = async (pluginId: number, state: string): Pr
 
   return state;
 };
+
+export const getLegacyLinkedInteractiveInfo = (embeddableRefId: string, laraData: ILaraData, callback: (info: ILegacyInteractiveState) => void) => {
```

Comment:
I used a callback to match the existing watchAnswer() method which is used in a useEffect() above the same call site as this function.  It also makes the useEffect() look nicer otherwise you have to use an immedatiely invoked function inside the useEffect function as they can't be passed async functions.